scaling_performance/network_optimization.adoc

```diff
@@ -170,6 +170,31 @@ network infrastructure.
 VXLAN-offload does not reduce latency. However, CPU utilization is reduced even
 in latency tests.
 
+[[scaling-performance-optimizing-mtu]]
+=== Optimizing the MTU for Your Network
+
+There are two MTUs (maximum transmission units) that matter: the network interface card (NIC) MTU, and the SDN overlay's MTU.
+
+The NIC MTU can be whatever you think is optimal for your network.
+
+However, the SDN overlay's MTU must be less than the NIC MTU by 50 bytes at a
+minimum. On a normal ethernet network, this should be set to around 1450. On a
+jumbo frame ethernet network, this should be around 8950.
+
+To configure the MTU, edit the node configuration file at *_/etc/origin/node/node-config.yaml_* and edit the following:
```

Comment:
@jeremyeder @knobunc @dcbw Would this be more at home in the [Recommended Install Practices section](https://docs.openshift.com/container-platform/3.5/scaling_performance/install_practices.html) then?

I've put in the other info.